inc/namespace.php

```diff
@@ -433,7 +434,7 @@ function get_or_create_wp_user( \OneLogin\Saml2\Auth $saml ) {
 	 *
```

Comment:
@Gazi96 Rafal, 
could you update the documentation block for the `wpsimplesaml_match_user` filter, to document the extra parameters, please?

Reply:
@mikelittle, good point, I have updated documentation.